### ISSUE TITLE
IDEA: select options that are on average 'more apart'

### ISSUE THREAD

**Issue by dennisschoenmakers** [quintel/etmobile#28]
Especially for the first questions, you'd rather give a higher change to selecting two options that are 'further apart', so they are rather simple to answer.

What do you think @antw?

**Comment by antw** [comment]
What might be nice is if we start with questions where the choices are quite different in their outcomes, and – as the visitor makes more correct answers – trend towards choices which are closer. i.e. start easy and get harder as the game progresses.

However, this probably requires that we have more choices. If the visitor got a couple of "easy" questions wrong they'd be left with only hard questions; this could get frustrating. Or maybe this isn't such a problem since we recycle choices which weren't selected?

Worth trying?

**Comment by dennisschoenmakers** [comment]
> Worth trying?

Absolutely! I think @ploh can have a go a creating an algorithm that weights in the difference in outcome, and recycle choices (but not too soon, you've just seen the effect! :smile: )
